cardano-api/src/Cardano/Api/ProtocolParameters.hs

```diff
@@ -676,6 +678,15 @@ instance FromCBOR ProtocolParametersUpdate where
         <*> fromCBOR
         <*> fromCBOR
 
+instance ToJSON ProtocolParametersUpdate where
+  toJSON =
+    genericToJSON defaultOptions{fieldLabelModifier = lowerFirst . drop 14}
```

Comment:
Explicit JSON instances are preferential. It makes it easier to debug round trip or golden test failures.